src/main/java/fi/aalto/cs/apluscourses/presentation/SelectStudentViewModel.java

```diff
@@ -33,6 +33,11 @@ public SelectStudentViewModel(@NotNull List<Student> students,
     this.authentication = authentication;
   }
 
+  /**
+   * Set the students.
+   *
+   * @param newStudents A list.  The list is copied once and no reference to the original list is maintained.
```

Comment:
Double space between "list." and "The"

Reply:
A stylistic choice to have double space after a full stop. (Okay, perhaps it wasn't intended here but it could have been!)